UISession 1/WeatherApplication.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Weather Application</title>
+</head>
+<body>
+
+
+	<!--Header-->
+	<header>
+		<div id="header_id" class="header">
+			<img src="images/footer-logo.png">
+			<div id="header_options" class="header-options">
+				<a href="home.html">Home</a>
+				<a href="features.html">Features</a>
+				<a href="reviews.html">Reviews</a>
+				<a href="download">Download</a>
+			</div>
+		</div>
+	</header>
+
+
+
+	<!--Cover-->
+	<div id="cover_head" class="cover-head">
+		<div id="cover_title" class="cover-title">
+			<h1>Get the most fun weather app</h1>
+			<a href = "download" id = "download">Download</a>
+			<a href = "features" id = "features">Features</a>
+		</div>
+		<div id="first_cover_image" class="cover-image">
+			<img src="images/phone1.png">
+		</div>
+	</div>
+
+	<div id="cover_features" class="cover-features">
+		<div id="cover_features_heading" class="cover-features-heading">
+			<h1>Perfect Features</h1>
+			<p>Only necessary</p>
+		</div>
+
+		<div id="cover_features_body" class="cover-features-body">
+			<div  class="cover-features-content">
+				<h2>Usablity</h2>
+				<p>Sometimes the simplest things are the hardest to find. So we created a new line for everyday life</p>
+			</div>
+			<div class="cove-features-content">
+				<h2>Parallax Effect</h2>
+				<p>Sometimes the simplest things are the hardest to find. So we created a new line for everyday life</p>
+			</div>
+			<div class="cover-features-content">
+				<h2>Unlimites Colors</h2>
+				<p>Sometimes the simplest things are the hardest to find. So we created a new line for everyday life</p>
+			</div>
+		</div>
+
+
+		<div id="cover_image_group" class="cover-image-group">
+			<div id="left_group_image" >
+				<img src="images/drag1.png">
+			</div>
+			<div id="center_left_group_image" >
+				<img src="images/drag3.png">
+			</div>
+			<div id="center_group_image" >
+				<img src="images/drag2.png">
+			</div>
+			<div id="center_right_group_image" >
+				<img src="images/drag1.png">
+			</div>
+			<div id="right_group_image" >
+				<img src="images/drag3.png">
+			</div>
+		</div>
+	</div>
+
+	<div id="cover_screenshot_header" class="cover-screenshot-header">
+		<h2=1>Screenshots</h1>
+		<p>The Brightest images</p>
+	</div>
+
+	<div id="first_cover_screenshot" class="cover-screenshot">

```

Comment:
avoid first,second..type of keyword. the id should be based on the content div contains.